.gitignore

```diff
@@ -10,3 +10,7 @@ logs/
 npm-debug.log
 
 config/config.proxy.js
+
+jsconfig.json
```

Comment:
Yes this is for typings flies that I have for VS Code. Some other people on our team have them too so I don't want them to get committed by accident ever.